examples/set-expression.yaml

```diff
@@ -0,0 +1,18 @@
+document:
+  dsl: '1.0.0'
+  namespace: test
+  name: set
+  version: '0.1.0'
+schedule:
+  on:
+    one:
+      with:
+        type: io.serverlessworkflow.samples.events.trigger.v1
+do: 
+  - initializeObject:
```

Comment:
Please remove the `initializeObject` from here, as the example is for the expression only.